cluster-setup.md

```diff
@@ -10,76 +19,121 @@ $ gcloud container clusters create gke-1 \
+2. Rename contexts
 
-```bash
-$ gcloud container clusters get-credentials gke-1 --zone us-west1-a
-Fetching cluster endpoint and auth data.
-kubeconfig entry generated for gke-1.
+    The prior step will have added credentials for your new clusters to your `kubeconfig`, but let's rename the contexts to something a little shorter:
 
-$ gcloud container clusters get-credentials gke-2 --zone us-east1-b
-Fetching cluster endpoint and auth data.
-kubeconfig entry generated for gke-2.
+    ```bash
 
-$ kubectl config rename-context ${GKE1_CONTEXT} gke-1
+    $ kubectl config rename-context gke_${PROJECT}_us-west1-a_gke-1 gke-1
 
-$ kubectl config rename-context ${GKE2_CONTEXT} gke-2
-```
+    $ kubectl config rename-context gke_${PROJECT}_us-east1-b_gke-2 gke-2
+    ```
 
-2. Enable the Hub, Anthos, and MultiClusterIngress APIs as done [here](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-for-anthos-setup#before_you_begin).
-3. [Register](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-for-anthos-setup#registering_your_clusters) your two clusters. Confirm that they are registered with Hub.
+3. Enable the Hub, Anthos, and MultiClusterIngress APIs for your GCP project as described [here](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-for-anthos-setup#before_you_begin).
 
-```
-$ gcloud container hub memberships list
-NAME   EXTERNAL_ID
-gke-1  50468ae8-29a3-4ea1-b7ff-0e216533619a
-gke-2  6c2704d2-e499-465d-99d6-3ca1f3d8170b
-```
+    ```bash
 
-4. Now enable Multi-cluster Ingress and specify `gke-1` as your config cluster.
+    $ gcloud services enable gkehub.googleapis.com
 
-```bash
-$ gcloud alpha container hub ingress enable \
-  --config-membership=projects/<your-project>/locations/global/memberships/gke-1
-```
+    $ gcloud services enable anthos.googleapis.com
 
-5. Confirm that MCI is configured properly.
+    $ gcloud services enable multiclusteringress.googleapis.com
+    ```
 
-```bash
-$ gcloud alpha container hub ingress describe
-createTime: '2020-11-14T20:50:53.856780163Z'
-featureState:
-  details:
-    code: OK
-    description: Ready to use
-  detailsByMembership:
-    projects/759444700240/locations/global/memberships/gke-1:
-      code: OK
-    projects/759444700240/locations/global/memberships/gke-2:
-      code: OK
-  hasResources: true
-  lifecycleState: ENABLED
-multiclusteringressFeatureSpec:
-  configMembership: projects/church-243723/locations/global/memberships/gke-1
-name: projects/church-243723/locations/global/features/multiclusteringress
-updateTime: '2020-11-14T20:50:54.761389487Z'
-```
+4. [Register](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-for-anthos-setup#registering_your_clusters) your two clusters (`gke-1` and `gke-2`). 
+
+    There are a few steps to complete as part of the registration process. A quick hint to get you going is the `gke-uri` for your GKE clusters. 
```

Comment:
Instead of showing the examples I would just recommend using the following command to get the URI:

`gcloud container clusters list --uri`